apisix/http/router/radixtree_host_uri.lua

```diff
@@ -125,7 +125,6 @@ local function create_radixtree_router(routes)
             })
         end
     end
-
```

Comment:
this change is not related to this PR. please revert it

Reply:
Thank you my bad